service.toml

```diff
@@ -36,7 +36,7 @@ optional = ["offset", "io_callback", "size"]
 optional = ["content_md5", "content_type", "io_callback", "storage_class", "server_side_encryption", "server_side_data_encryption", "server_side_encryption_key_id"]
 
 [namespace.storage.op.create_append]
-optional = ["content_type", "server_side_encryption", "server_side_encryption_key_id", "server_side_data_encryption", "storage_class"]
```

Comment:
Why are they removed?

Reply:
According to [aliyun API reference](https://help.aliyun.com/document_detail/31981.html), apend objects stored in the OSS server cannot be encrypted by using a specified CMK ID. And the request headers of `AppendObject` do not include `x-oss-server-side-data-encryption` and `x-oss-server-side-encryption-key-id`. So I removed the pairs.